src/app/components/Curation/index.test.tsx

```diff
@@ -109,7 +109,7 @@ describe('Curation', () => {
         {
           toggles: {
             mostRead: { enabled: true },
-            radioSchedule: { enabled: true },
+            onDemandRadioSchedule: { enabled: true },
```

Comment:
I am wondering how the homepage works & which version of the radioSchedule toggle it uses... 🤔 I might need to do a bit of digging to find out - could I get some help with this whenever you have time?

Just in case, could we keep both radioSchedule & onDemandRadioSchedule?

 ```suggestion
            radioSchedule: { enabled: true },
            onDemandRadioSchedule: { enabled: true },
```